src/util/requestUtils.ts

```diff
@@ -42,6 +42,7 @@ export interface WSClientConfigurableOptions {
   reconnectTimeout?: number;
   restOptions?: RestClientOptions;
   requestOptions?: AxiosRequestConfig;
+  reconnect?: boolean;
```

Comment:
```suggestion
  reconnectOnClose?: boolean;
```

Reply:
resolved in https://github.com/kenchambers/ftx-api/commit/383c18eef50b2d411d3ad4a6190aa3c3b53f5b01

@tiagosiebler

Reply:
This one still needs to be renamed to `reconnectOnClose`.

Reply:
oops 5d5211b

Reply:
my bad!